nsxt/resource_nsxt_policy_transit_gateway.go

```diff
@@ -46,6 +55,136 @@ var transitGatewaySchema = map[string]*metadata.ExtendedSchema{
 			SdkFieldName: "TransitSubnets",
 		},
 	},
+	"is_default": {
+		Schema: schema.Schema{
+			Type:     schema.TypeBool,
+			Optional: true,
+		},
+		Metadata: metadata.Metadata{
+			SchemaType:   "bool",
+			SdkFieldName: "IsDefault",
+		},
+	},
+	"high_availability_config": {
+		Schema: schema.Schema{
+			Type:     schema.TypeList,
+			MaxItems: 1,
+			Elem: &metadata.ExtendedResource{
+				Schema: map[string]*metadata.ExtendedSchema{
+					"ha_mode": {
+						Schema: schema.Schema{
+							Type:         schema.TypeString,
+							ValidateFunc: validation.StringInSlice(transitGatewayHaModeValues, false),
+							Optional:     true,
+							Default:      model.TransitGatewayHighAvailabilityConfig_HA_MODE_ACTIVE,
+						},
+						Metadata: metadata.Metadata{
+							SchemaType:   "string",
+							SdkFieldName: "HaMode",
+						},
+					},
+					"edge_cluster_paths": {
+						Schema: schema.Schema{
+							Type: schema.TypeList,
+							Elem: &metadata.ExtendedSchema{
+								Schema: schema.Schema{
+									Type:         schema.TypeString,
+									ValidateFunc: validatePolicyPath(),
+								},
+								Metadata: metadata.Metadata{
+									SchemaType: "string",
+								},
+							},
+							Optional: true,
+						},
+						Metadata: metadata.Metadata{
+							SchemaType:   "list",
+							SdkFieldName: "EdgeClusterPaths",
+						},
+					},
+				},
+			},
+			Optional: true,
+		},
+		Metadata: metadata.Metadata{
+			SchemaType:   "struct",
+			SdkFieldName: "HighAvailabilityConfig",
+			ReflectType:  reflect.TypeOf(model.TransitGatewayHighAvailabilityConfig{}),
+		},
+	},
+	"span": {
+		Schema: schema.Schema{
+			Type:     schema.TypeList,
+			MaxItems: 1,
+			Elem: &metadata.ExtendedResource{
+				Schema: map[string]*metadata.ExtendedSchema{
+					"cluster_based_span": {
+						Schema: schema.Schema{
+							Type:     schema.TypeList,
+							MaxItems: 1,
+							ExactlyOneOf: []string{
+								"span.0.cluster_based_span",
+								"span.0.zone_based_span",
+							},
+							Elem: &metadata.ExtendedResource{
+								Schema: map[string]*metadata.ExtendedSchema{
+									"span_path": {
+										Schema: schema.Schema{
+											Type:         schema.TypeString,
+											Required:     true,
+											ValidateFunc: validatePolicyPath(),
+										},
+										Metadata: metadata.Metadata{
+											Skip: true,
+										},
+									},
+								},
+							},
+						},
+						Metadata: metadata.Metadata{
+							Skip: true,
+						},
+					},
+					"zone_based_span": {
```

Comment:
the schema is missing use_all_zones

Reply:
`use_all_zones` is marked as readonly, we do not include those normally.